### ISSUE TITLE
[Config Support]: Invalid data found when processing input

### ISSUE THREAD

**Issue by gravelfreeman** [blakeblackshear/frigate#9048]
### Describe the problem you are having

My `camera-2` and `camera-3` are throwing out errors in the logs and won't initiate record.

### Version

0.12.1-367d724

### Frigate config file

```yaml
mqtt:
  enabled: True
  host: core-mosquitto.local.hass.io
  port: 1883
  user: "***"
  password: "***"

database:
  path: /config/frigate.db

go2rtc:
  streams:
    camera_2: 
      - "ffmpeg:http://***/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=***#video=copy#audio=copy#audio=opus"
    camera_2_sub: 
      - "ffmpeg:http://***/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=***"
    camera_3: 
      - "ffmpeg:http://***/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=***#video=copy#audio=copy#audio=opus"
    camera_3_sub: 
      - "ffmpeg:http://***/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=***"

cameras:
  camera-1:
    ffmpeg:
      inputs:
        - path: rtsp://admin:***@***/h264Preview_01_main
          roles:
            - record
        - path: rtsp://admin:***@***/h264Preview_01_sub
          roles:
            - detect
            - rtmp
    detect:
      width: 1536
      height: 576
  camera-2:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/camera_2?video&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/camera_2_sub?video
          input_args: preset-rtsp-restream
          roles:
            - detect
  camera-3:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/camera_3?video&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/camera_3_sub?video
          input_args: preset-rtsp-restream
          roles:
            - detect

record:
  enabled: True
  retain:
    days: 7
    mode: motion
  events:
    retain:
      default: 14
      mode: active_objects
```

### Relevant log output

```shell
2023-12-22 00:23:40.200834213  19:23:40.200 ERR [exec] timeout url="exec:ffmpeg -hide_banner -v error -fflags nobuffer -flags low_delay -i http://***/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=***-map 0:v:0? -c:v:0 copy -map 0:a:0? -c:a:0 copy -map 0:a:0? -c:a:1 libopus -ar:a:1 48000 -ac:a:1 2 -user_agent ffmpeg/go2rtc -rtsp_transport tcp -f rtsp rtsp://localhost:8554/***"

2023-12-22 00:23:40.201099286  19:23:40.200 ERR github.com/AlexxIT/go2rtc/cmd/streams/producer.go:53 > error=timeout url=ffmpeg:http://10.0.10.52/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=***#video=copy#audio=copy#audio=opus
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Any other information that may be helpful

I can access the camera feeds in browser with these URLs;

`http://***/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=***#video=copy#audio=copy#audio=opus `

`http://***/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=***`

**Comment by NickM-27** [comment]
I would suggest updating go2rtc https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

**Comment by gravelfreeman** [comment]
> I would suggest updating go2rtc https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

I've followed the instructions on the link you've shared. I think it have worked since I can see this in Frigate's logs;

```python
2023-12-22 15:52:34.715039221  10:52:34.714 INF go2rtc version 1.8.4 linux/386
``` 

The permissions are `rwxr-xr-x` on the renamed `go2rtc` file in config folder.

Unfortunately I'm getting the same errors as in my orignal post.

**Comment by NickM-27** [comment]
Okay, a full copy of frigate and go2rtc logs would be helpful

**Comment by NickM-27** [comment]
Cameras are timing out

**Comment by gravelfreeman** [comment]
> Cameras are timing out

What does it mean and how can I figure this out?

**Comment by NickM-27** [comment]
It means the stream of video / audio data from the camera is being broken, this can be a camera issue, network issue, etc.  with reolink cameras sometimes it is just a camera issue due to their buggy nature. Are these cameras connected via WiFi or PoE?

**Comment by gravelfreeman** [comment]
> It means the stream of video / audio data from the camera is being broken, this can be a camera issue, network issue, etc. with reolink cameras sometimes it is just a camera issue due to their buggy nature. Are these cameras connected via WiFi or PoE?

They're connected by POE and they doesn't time out in the Reolink application or in a browser.

**Comment by NickM-27** [comment]
That's not a surprise though, reolink uses a proprietary protocol on the browser so it's different than streaming via rtsp or http-flv. They also don't report errors in the stream like frigate doesn't

**Comment by gravelfreeman** [comment]
> That's not a surprise though, reolink uses a proprietary protocol on the browser / app so it's different than streaming via rtsp or http-flv. They also don't report errors in the stream like frigate doesn't

I was able to setup them up without errors using RTSP. I know it's not recommended but it works. I can watch the videos on the companion app on Android which was the most important thing when I'm not home. I'll try to setup the push notifications and hope h265 will work on there as well.

Also I noticed that everywhere it says to not use the CPU option. It's running on a server with 36 cores and 72 threads. The vm doesn't have full host ressources and right now it's idle at 50% cpu usage since I installed Frigate. My server CPU is at 10% usage. Still I'm running other vms and over 50 kubernetes containers. It might be different once I setup the cameras outside. Right now they're in a closet and I guess there's not much processing done with it at the moment.
